app/Http/Controllers/PostController.php

```diff
@@ -1,148 +0,0 @@
-<?php
-
-namespace App\Http\Controllers;
-
-use App\Category;
-use App\Tag;
-use App\User;
-use Illuminate\Http\Request;
-use App\Post;
-
-class PostController extends Controller
-{
-
-    public function __construct()
-    {
-        $this->middleware('auth');
-    }
-
-    /**
-     * Display a listing of the resource.
-     *
-     * @return \Illuminate\Http\Response
-     */
-    public function index()
-    {
-        $this->authorize('viewAny', Post::class);
-
-        $posts = Post::orderBy('id', 'DESC')->paginate(5);
```

Comment:
you can also use the `latest()` scope instead of orderBy('id', 'DESC'). It orders by, I believe, the created_at column desc.